pkg/logforwarding/interactive.go

```diff
@@ -0,0 +1,156 @@
+package logforwarding
+
+import (
+	"fmt"
+	"maps"
+	"slices"
+	"strings"
+
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/ocm"
+)
+
+// Options for initial interactive prompt
+const skip = "Skip"
+const cloudWatch = "CloudWatch"
+const s3 = "S3"
+const both = "Both"
+
+// Options for CW pod groups
+var podGroupsOptions = map[string]string{
```

Comment:
we have an API endpoint to get the allowed groups and applications; it is accessible via "client.ocm.ClustersMgmt().V1().LogForwarding().Groups().List()" and "client.ocm.ClustersMgmt().V1().LogForwarding().Groups().List()" in the SDK